src/views/DidDowngrade/DidDowngrade.tsx

```diff
@@ -3,14 +3,12 @@ import { Link } from 'react-router-dom';
 import BN from 'bn.js';
 import { browser } from 'webextension-polyfill-ts';
 import { IDidDetails } from '@kiltprotocol/types';
-
-import { DidChain } from '@kiltprotocol/did';
+import { IChainDeposit } from '@kiltprotocol/did/lib/cjs/Did.chain';
```

Comment:
Importing from paths like this one is not good, like accessing some private property. I’ll show a better alternative below.